typedb/simulation.gql

```diff
@@ -84,7 +91,9 @@ region sub entity, abstract,
 
 continent sub region;
 country sub region,
-    owns currency;
+    owns currency,
+    plays nationality:state,
+    plays citizenship:state;
```

Comment:
Would `country` not be a better role than `state` here?

Reply:
Yes let's do that